model-optimizer/extensions/front/tf/ObjectDetectionAPI.py

```diff
@@ -486,6 +486,23 @@ def nodes_to_remove(self, graph: Graph, match: SubgraphMatch):
                 new_nodes_to_remove.remove(node_to_keep)
         return new_nodes_to_remove
 
+    def is_preprocessing_applied_before_resize(self, to_float: Node, mul: Node, sub: Node):
+        """
+        The function checks if the output of 'to_float' operation is consumed by 'mul' or 'sub'. If this is true that
+        means that the pre-processing (mean/scale) is applied before the image resize. The image resize was applied
```

Comment:
If this is true that means that the pre-processing (mean/scale) is applied before the image resize. ->
If this is true, the pre-processing (mean/scale) is applied before the image resize.

Reply:
Fixed